ports/stm32/boards/ARDUINO_PORTENTA_H7/board_init.c

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Please run this file through uncrustify, via `./tools/codeformat.py ports/stm32/boards/ARDUINO_PORTENTA_H7/board_init.c`.